### PR TITLE
PyDT - General Python transformer

### PR DESCRIPTION
**Pull request by mhouston007**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Python alternative to the DT transformer

## Screenshots
image

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/PyDT/PyDT.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+def py_dt(val):
+    source = 'return_results({})'.format(demisto.args().get('Python', "Python arg not found"))
+    code = compile(source, '<string>', 'exec')
```
Comment (anara123) [inline Packs/CommonScripts/Scripts/PyDT/PyDT.py:6]: We can't allow arbitrary user input code execution.

**Review by anara123 — CHANGES_REQUESTED** [review]

**Comment by anara123** [comment]
Hello @mhouston007 
Thank you for contribution. But unfortunately I must reject it. We can't allow arbitrary user input code execution. 
If you have other way to implement it, feel free to reopen the PR and update the code.
Closing for now

**Comment by anara123** [comment]
@mhouston007 Sorry, Didn't noticed you have other files in your PR. Please remove the PyDT script or change the implementation,

**Comment by reutshal** [comment]
@anara123 The other files are being reviewed in this PR by @barchen1 : https://github.com/demisto/content/pull/8659/files

**Comment by mhouston007** [comment]
Didn't mean to include the other files, those are a separate PR, closing this one

**Comment by mhouston007** [comment]
Made the updates a bit too restricted, closing until I can get it right

**Pull request closed without merging**